module/x/gravity/types/msgs.go

```diff
@@ -577,9 +578,9 @@ func (msg *MsgCancelSendToEth) GetSignBytes() []byte {
 
 // GetSigners defines whose signature is required
 func (msg *MsgCancelSendToEth) GetSigners() []sdk.AccAddress {
-	acc, err := sdk.ValAddressFromBech32(msg.Sender)
+	acc, err := sdk.AccAddressFromBech32(msg.Sender)
```

Comment:
This changes the original behavior, the bech32 prefixes are different for these two addresses. We should double-check how txs are submitted here to make sure it's correct.

Reply:
ideally we standardize this behavior across the app. I'm in favor of only using the accaddress prefix for validators

Reply:
i fixed this on my refactor